### PR TITLE
DBZ-3984 Mysql-Connector fails parsing invalid decimal format DDL statement

### PR DESCRIPTION
**Pull request by harveyyue**
fix bug [DBZ-3984](https://issues.redhat.com/browse/DBZ-3984)

### CODE REVIEW

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/listener/ColumnDefinitionParserListener.java [hunk 1]
```diff
@@ -193,7 +194,19 @@ else if (dataTypeContext instanceof MySqlParser.DimensionDataTypeContext) {
 
             if (dimensionDataTypeContext.lengthTwoOptionalDimension() != null) {
                 List<MySqlParser.DecimalLiteralContext> decimalLiterals = dimensionDataTypeContext.lengthTwoOptionalDimension().decimalLiteral();
-                length = Integer.valueOf(decimalLiterals.get(0).getText());
+                if (decimalLiterals.get(0).REAL_LITERAL() != null) {
+                    String[] digits = decimalLiterals.get(0).getText().split("\\.");
```
Comment (gunnarmorling) [inline debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/listener/ColumnDefinitionParserListener.java:198]: Could you extract a `Pattern` and store it as a static field, avoiding the repeated parsing of the regexp?

**Review by gunnarmorling — APPROVED** [review]
Wow, MySQL never disappoints :) Apart from the commit authorship fix, one small comment inline. And can you also send a PR against the upstream Antlr grammar (e.g. like this one: https://github.com/antlr/grammars-v4/pull/2169), as we try to keep the two in sync as much as we can. Thx!

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/listener/ColumnDefinitionParserListener.java (hunk 1 above)
Comment (harveyyue) [reply debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/listener/ColumnDefinitionParserListener.java:198]: @gunnarmorling
I have fixed these issue with new pr:
https://github.com/debezium/debezium/pull/2692

**Comment by harveyyue** [comment]
refer #2692

**Pull request closed without merging**